strings/base_collections_map.h

```diff
@@ -66,3 +66,36 @@ WINRT_EXPORT namespace winrt
         return make<impl::observable_map<K, V, std::unordered_map<K, V, Hash, KeyEqual, Allocator>>>(std::move(values));
     }
 }
+
```

Comment:
I wasn't sure whether to put it here or in `base_collections.h`. Feel free to edit.

Reply:
Works for me. 👍